ios-apps-master/Beer_Diary/Beer_Diary/Filter.swift

```diff
@@ -0,0 +1,145 @@
+//
+//  Filter.swift
+//  Beer_Diary
+//
+//  Created by Piotr Mielcarzewicz on 13/11/16.
+//  Copyright © 2016 Piotr Mielcarzewicz. All rights reserved.
+//
+
+import UIKit
+
+var filterArray = [String]()
+var filterby = "none"
```

Comment:
I would make the `filterby` variable an enum, so you can filter by a constant amount of things, not arbitrary string